modules/calib3d/src/calibration_handeye.cpp

```diff
@@ -449,6 +462,9 @@ static void calibrateHandEyeHoraud(const std::vector<Mat>& Hg, const std::vector
             Mat Rcij = Hcij(Rect(0, 0, 3, 3));
 
             Mat qgij = rot2quat(Rgij);
+            if (qgij.at<double>(0, 0) < 0) {
+                qgij *= -1;
```

Comment:
Indeed, this can be checked with this online tool: https://www.andre-gaschler.com/rotationconverter/